deafrica/data/chirps.py

```diff
@@ -45,7 +51,10 @@ def download_and_cog_chirps(
         # Set up a daily process
         in_file = f"chirps-v2.0.{year}.{month}.{day}.tif.gz"
         in_href = DAILY_URL_TEMPLATE.format(year=year, in_file=in_file)
-        in_data = f"/vsigzip//vsicurl/{in_href}"
+        if not check_for_url_existence(in_href):
+            in_file = f"chirps-v2.0.{year}.{month}.{day}.tif"
+            in_href = DAILY_URL_TEMPLATE.format(year=year, in_file=in_file)
+        in_data = f"/vsicurl/{in_href}"
```

Comment:
I don't think you can change the order of these. Can you please add a test, or at least test locally.

Reply:
Yep, test is failing @dominicschaff. Please ensure your changes have coverage :-)

Reply:
Sorry, I was thinking clearly. I will duplicate the tests and have both types

Reply:
I fixed this, and added tests that are on a different date. I also added a new docker-compose service that can run the tests. I didn't want to change the current test method however, just to make sure that it still works